core/src/main/java/com/netflix/conductor/service/WorkflowValidatorImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.conductor.service;
+
+import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.execution.ValidationException;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+
+import java.util.List;
+
+
+public class WorkflowValidatorImpl implements WorkflowValidator {
+	@Override
+	public void validate(WorkflowDef workflowDef, Workflow workflow) {
+		if (workflowDef.getInputDefinition().isEmpty()) {
+			return;
+		}
+		final JSONObject jsonSchema = new JSONObject(workflow.getInput());
+		final JSONObject jsonSchemaDefinition = new JSONObject(workflowDef.getInputDefinition());
+		final Schema schema = SchemaLoader.builder().draftV7Support()
```

Comment:
Could the latest draft be used instead? (See in https://json-schema.org/, `16 September 2019: Draft 2019-09 (formerly known as draft-08) has been published!`)

Reply:
I think most toolset still only support up to draft-07, having said that, the code could be changed in the future to support more than one version.

Reply:
Indeed, as @elisherer already explained, draft-08 is not supported by this library (yet)